Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -239,8 +239,7 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
 
     func bindEventListener() {
         self.eventListenerLoop = Task { [weak self, weak channel] in
-            guard let channel else { return }
-            let eventListener = channel.eventStream()
+            let eventListener = channel!.eventStream()
```

Comment:
Try changing `weak channel` to `unowned channel`, it should have the same effect as force unwrapping the weak channel.

Reply:
I just tried: 

```swift
self.eventListenerLoop = Task { [weak self, unowned channel] in
            guard let channel else { return }
            let eventListener = channel.eventStream()
```

In the graph view this creates an owning reference to the channel that is retained.

should I commit

```swift
self.eventListenerLoop = Task { [weak self, unowned channel] in
            let eventListener = channel!.eventStream()
```

instead?

Reply:
Yeah remove the guard statement